### ISSUE TITLE
Error installing SimPEG - Geoana

### ISSUE THREAD

**Issue by jc-barreto** [simpeg/simpeg#739]

**Comment by jc-barreto** [comment]
Hi, I have SimPEG at university and now I'm trying to install in my particular pc.
I did the same steps: 1º Anaconda, 2º Visual Basic, 3º Upgrade pip, 4º install SimPEG by cmd line, and I got the error : 
![image](https://user-images.githubusercontent.com/42352209/48845656-fff52a00-ed94-11e8-9659-0e6029b69c35.png)

So, I tried to install through git
and I got another error
![image](https://user-images.githubusercontent.com/42352209/48845695-19967180-ed95-11e8-98af-7b003c997d97.png)

I don't know what I did differently from the first time I installed... 
Could someone help me?

**Comment by lheagy** [comment]
Hi @jc-barreto, a couple things to try, can you first try running 
```
pip install SimPEG
``` 
(e.g. without the python -m)

If that fails, then please try pip installing `geoana` and then try SimPEG again. 

Please let us know if that works!

**Comment by jc-barreto** [comment]
Hello again,

I tried the pip install SimPEG and I got the same error.
so I did pip install geona and after pip install SimPEG, now I'm having a 
problem to build wheel for discretize, but I have installed the Visual C++ 2017, I should have the MSVCCompiler...
I don't know

![image](https://user-images.githubusercontent.com/42352209/49254463-2bc18100-f421-11e8-81cc-4e35f66e6d0a.png)


![image](https://user-images.githubusercontent.com/42352209/49254510-4dbb0380-f421-11e8-9534-9785540af44f.png)


![image](https://user-images.githubusercontent.com/42352209/49254572-73e0a380-f421-11e8-802b-5019a7e77a4f.png)

![image](https://user-images.githubusercontent.com/42352209/49254601-86f37380-f421-11e8-9ad9-f0a2c153de37.png)

**Comment by lheagy** [comment]
👋 Hi @jc-barreto: were you able to get up and running?

**Comment by jc-barreto** [comment]
Yeah... I got it, but I don't know exactly how. I removed all visual basic from my pc (I had a lot of old versions packages besides 2017), and then I installed just 2017.
